chapters/emergencystop.adoc

```diff
@@ -0,0 +1,18 @@
+== Emergency stop
+
+This rule only applies to Division A.
+
+.Definition
+
+A team can ask to stop the game immediately after a grace period of 10 seconds or at the next stoppage, whichever happens first regardless of the current situation. It will receive a yellow card for this and must take a timeout immediately. If the team is out of timeouts, it is still allowed to remove robots from the field, but can not use any remaining timeout time.
```

Comment:
If I read this correctly, this Emergency stop starts a timeout for the team.

I'm pretty sure - although I cannot find it right now - that a time (in stop state) always ended with the previous command, either indirect or direct for yellow or blue.

Maybe a clarification here (and in #10) how to continue the game after the forced timeout without a resolution like in the stop-state version. How did we handle Yellow cards before #10, when they stopped a running game?

Reply:
Good point. A regular timeout can only be taken during stop, so the game could be continued normally afterwards. Now, there are three possible situations:

1. The 10 seconds run off and the game is stopped.
2. The game is stopped before the 10 seconds run off because the ball left the field, a foul occurred, etc.
3. The human ref stopped the game manually before the 10 seconds run off

Suggested Solution 1:
For 2, we continue just like with a regular timeout.
For 1 and 3, we start with a a) force start or with a b) free kick for the opposing team.

Suggested Solution 2:
We continue with either a) force start or b) free kick for all three situations.

I tend towards solution 1b). Otherwise, the opposing team could do a foul within the 10s that is not punished with a freekick.

Regarding the yellow cards: I think, a yellow card should have been given only during stop. At least, that was the assumption, but I remember that there were inconsistencies in the GC in this regard, because collisions could cause fouls which in turn could result in multiple fouls violation which lead to a yellow card. Btw: The rule change proposal already specifies what to do in the case, the robots are not removed in time.

Reply:
#10 states that "in 2020 no penalty will be given to the team [...] ". So there's no description how to continue the game after the game-state has been left forcefully due to 10 seconds running off the clock.

Reply:
I'd go for 1a, but I think anything in Solution 1 sounds fair. I just want it written down in the rules.

Reply:
Just added these procedures to restart the game

> Suggested Solution 1:
>For 2, we continue just like with a regular timeout.
>For 1 and 3, we start with a ~~a) force start or with a~~ b) free kick for the opposing team.